pkg/operator/mon/mon_test.go

```diff
@@ -234,8 +234,9 @@ func TestSaveMonEndpoints(t *testing.T) {
 	c.clusterInfo.Monitors["mon1"].Endpoint = "2.3.4.5:6790"
 	c.maxMonID = 2
 	c.mapping.Node["mon1"] = &NodeInfo{
-		Name:    "node0",
-		Address: "1.1.1.1",
+		Name:     "node0",
+		Address:  "1.1.1.1",
+		Hostname: "",
```

Comment:
can we add a test where the Hostname is not empty? That way we can test this path.

Reply:
Sure! WIll act accordingly. Thx for the feedback! :)